Customer.io project/components/Sidebar.tsx

```
@@ -0,0 +1,181 @@
+          <div>
+            <h3 className="font-medium text-sm text-gray-900 mb-3">
+              Templates
+            </h3>
+            <div className="mt-2 space-y-1">
+              {menuItems.map((item, index) => (
+                <React.Fragment key={item.name}>
+                  <button
+                    onClick={() => onInsertBlock(item.name.toLowerCase())}
+                    className="flex items-center space-x-3 w-full p-2 text-left hover:bg-blue-50 rounded-lg group"
+                  >
+                    <div className="w-8 h-8 bg-gray-100 rounded-lg flex items-center justify-center group-hover:bg-blue-100">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)